main/templates/main/base.html

```diff
@@ -1,23 +1,21 @@
-{% load static %}
-{% load django_bootstrap5 %}
-<!DOCTYPE html>
+{% load static %} {% load django_bootstrap5 %}
```

Comment:
I'd suggest to revert running prettrier and add a few tags to tell it to ignore some parts. This headers just look much better if each is int its line. So, leave this as:

```suggestion
<!-- prettier-ignore -->
{% load static %}
{% load django_bootstrap5 %}
```

And the same with the other HTML files. These `{%...` lines are just not understood by prettier as HTML because they are django stuff. Check this commit in Paricia where I've done the change and avoided the reformatting of the django tags. 

https://github.com/ImperialCollegeLondon/paricia/pull/350/commits/8357eeced35cdba04168766c82724d52f22ca995